src/Jackalope/ImportExport/ImportExport.php

```diff
@@ -95,7 +95,7 @@ public static function importXML(NodeInterface $parentNode, NamespaceRegistryInt
         }
 
         $xml = new FilteredXMLReader;
-        $xml->open($uri);
+        $xml->open($uri, null, LIBXML_PARSEHUGE);
```

Comment:
according to http://php.net/manual/en/libxml.constants.php this exists since php 5.3.2 which is fine, but only with libxml 2.7+ . can you please add a version check and only add this if the libxml version is 2.7+?